src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -444,6 +448,26 @@ protected function getEntitiesByModule()
         }
         return $entitiesByModule;
     }
+    
+    /**
+     * Loads all modules and themes installed, including app. Uses the format of
+     * the @link ModuleLoader manifest for themes as well.
+     * Themes can be references with "themes/{theme name}".
+     */
+    private function getModulesAndThemes(): array
+    {
+        $modules = ModuleLoader::inst()->getManifest()->getModules();
+        // load themes as modules
+        $themes = array_diff(scandir(THEMES_PATH), ['..', '.']);
+        if ($themes) {
+            foreach ($themes as $theme) {
+                if (is_dir(Path::join(THEMES_PATH, $theme))) {
+                    $modules['themes/' . $theme] = new Module(Path::join(THEMES_PATH, $theme), BASE_PATH);
```

Comment:
Been thinking a bit more about this - we should probably use some delimiter we _know_ won't be used in an actual module name. Colons aren't allowed in composer names so this protects against the unlikely edge case of a module called `themes/my-module`

```suggestion
                    $modules['themes:' . $theme] = new Module(Path::join(THEMES_PATH, $theme), BASE_PATH);
```

Probably also makes sense to set `'themes:'` as a private const on this class, since it's referenced a few times and this makes it super clear that those usages do have to explicitly be the same value.
```php
private const THEME_PREFIX = 'themes:';
```

```suggestion
                    $modules[self::THEME_PREFIX . $theme] = new Module(Path::join(THEMES_PATH, $theme), BASE_PATH);
```

Reply:
done.